src/tools/miri/tests/fail/stacked_borrows/drop_in_place_retag.rs

```diff
@@ -3,16 +3,9 @@
 
```

Comment:
Might be worth indicating in the test name and/or comment that this is specifically about `drop_in_place` on a type that does not need dropping.